scripts/generate-cocoa-bindings.ps1

```diff
@@ -47,23 +84,30 @@ Write-Output "iPhoneSdkVersion: $iPhoneSdkVersion"
 #     `#import "SomeHeader.h"`
 # This causes sharpie to fail resolve those headers
 $filesToPatch = Get-ChildItem -Path "$CocoaSdkPath/Headers" -Filter *.h -Recurse | Select-Object -ExpandProperty FullName
-foreach ($file in $filesToPatch) {
-    if (Test-Path $file) {
+foreach ($file in $filesToPatch)
```

Comment:
Everything from here on is just reformatting (nothing functional)

Reply:
question(style): we are not 100 % consistent with indentation style in PowerShell, from what I can glean
- most of `*.ps1` is in _Allman_ style
- only some of `*.ps1` is in _Kernighan & Ritchie_ style

Similar siutation in `sentry-powershell` and `sentry-unity`.

I believe that `dotnet/runtime`, other Microsoft projects, and the community in large prefers _K&R_ style for _PowerShell_.

Question:
Do we prefer _Allman_ style in general for all languages in `sentry-dotnet`, `sentry-unity` and `sentry-powershell`?

Reply:
I've created https://github.com/getsentry/sentry-dotnet/issues/4409 to tackle this.